Manifest.toml

```diff
@@ -154,22 +139,10 @@ uuid = "ca575930-c2e3-43a9-ace4-1e988b2c1908"
 deps = ["Artifacts", "CompilerSupportLibraries_jll", "Libdl"]
 uuid = "4536629a-c528-5b80-bd46-f80d51c5b363"
 
-[[deps.Parsers]]
```

Comment:
Do we really want to track the changes in the Manifest. To be honest I don't care about changes for many of the packages..